packages/models-library/src/models_library/api_schemas_webserver/functions_wb_schema.py

```diff
@@ -0,0 +1,276 @@
+from collections.abc import Mapping
+from enum import Enum
+from typing import Annotated, Any, Literal, TypeAlias
+from uuid import UUID
+
+from common_library.errors_classes import OsparcErrorMixin
+from models_library import projects
+from models_library.services_types import ServiceKey, ServiceVersion
+from pydantic import BaseModel, Field
+
+from ..projects import ProjectID
+
+FunctionID: TypeAlias = UUID
+FunctionJobID: TypeAlias = UUID
+FileID: TypeAlias = UUID
+
+InputTypes: TypeAlias = FileID | float | int | bool | str | list
+
+
+class FunctionSchemaClass(str, Enum):
+    json_schema = "application/schema+json"
+
+
+class FunctionSchemaBase(BaseModel):
+    schema_content: Any = Field(default=None)
```

Comment:
how does this type go throught the linter? it should be 
```suggestion
    schema_content: Any | None = Field(default=None)
```

Reply:
Fixed (Any can't be None?)